src/ChannelServer/Util/GmCommands.cs

```diff
@@ -488,7 +489,16 @@ private CommandResult HandleClearInventory(ChannelConnection conn, Character sen
 
 			return CommandResult.Okay;
 		}
-	}
+
+		private CommandResult HandleTestHealSkill(ChannelConnection conn, Character sender, Character target, string command, string[] args)
```

Comment:
I think we shouldn't add a test command for every little thing, better use the test command that's already there, and don't commit the changes to it.